cartography/intel/gcp/crm.py

```diff
@@ -0,0 +1,47 @@
+# Google Compute Resource Manager
+# https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy
+
+import googleapiclient.discovery
+import logging
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def get_gcp_organizations(credentials):
+    # See https://github.com/googleapis/google-api-python-client/issues/299#issuecomment-268915510 and related issues
+    # on why we set cache_discovery=False to suppress scary-looking warnings.
+    crm = googleapiclient.discovery.build('cloudresourcemanager', 'v1', credentials=credentials, cache_discovery=False)
```

Comment:
Separate service creation from use.

Reply:
Think I should move the `discovery.build()` call to `sync_organizations()` or to `start_gcp_ingestion()`?

Additional background: we will need to make additional calls to `discovery.build()` to create other `Resource` objects to pull other objects that don't fall under the Compute Resource Manager (e.g. Compute).

Reply:
I'm kinda leaning toward `sync_organizations` since `start_*_ingestion()` functions seem to only handle cred initialization

Reply:
I would suggest factoring it into its own method, then calling it in `sync_organizations` and passing the service object to `get_gcp_organizations`.